crates/optimism/evm/src/error.rs

```diff
@@ -21,10 +21,19 @@ pub enum OpBlockExecutionError {
     /// Thrown when a database account could not be loaded.
     #[error("failed to load account {_0}")]
     AccountLoadFailed(alloy_primitives::Address),
+    /// Thrown when a L1 block execution failed.
+    #[error("execution error on L1: {_0}")]
+    Eth(BlockExecutionError),
```

Comment:
```suggestion
    #[error(transparent)]
    Eth(BlockExecutionError),
```
the commit message "execution error on L1" is wrong, because the error is not occurring on l1. it's an error occurring on l2, that also can occur on l1 (as opposed to for example `L1BlockInfoError` which cannot occur on l1). does that make sense?